src/PsrTarget.php

```diff
@@ -19,16 +19,7 @@ class PsrTarget extends Target implements LoggerAwareInterface
 {
     use LoggerAwareTrait;
 
-    /**
-     * @var bool If enabled, logger use original timestamp from buffer
-     * @since 1.1.0
-     */
-    public $addTimestampToContext = false;
-
-    /**
-     * @var array
-     */
-    private $_levels = [
+    const LEVELS = [
```

Comment:
Won't work with PHP 5.4 and we're supporting it. I think this part should be reverted.

Reply:
Ok, but we need additional level map, because $this->_levels contains only user specified levels or all levels if not specified